Automattic/wp-calypso
# Disable agency dashboard down time email recipient setting

Disable the agency dashboard notification email recipient settings to make the value read only

#### Proposed Changes

* This PR sets the value of the users email for monitor notifications to be disabled. 

#### Testing Instructions



* Run `git checkout update/disable-agency-dashboard-notification-email`
* Run `yarn-start-jetpack-cloud`
* Visit the dashboard at http://jetpack.cloud.localhost:3000/dashboard
* Click on the clock icon and ensure that if the user has email notifications turned on the value cannot be changed and is disabled.
* 
<img width="452" alt="Screenshot 2023-01-12 at 7 18 00 AM" src="https://user-images.githubusercontent.com/1273880/212090406-b0d856cf-61e0-4bbc-8dd3-40559dd715e8.png">



#### Pre-merge Checklist





Related to #
